webapp/src/dogma/features/auth/authSlice.ts

```diff
@@ -53,6 +53,19 @@ export const login = createAsyncThunk('/auth/login', async (params: LoginParams,
   return false;
 });
 
+export const logout = createAsyncThunk('/auth/logout', async () => {
```

Comment:
After the logout process is fulfilled, the redux store needs to be updated.
https://redux-toolkit.js.org/api/createAsyncThunk
`logout` can be added to `extraReducers` in `authSlice`. 
For example:
```ts
.addCase(logout.fulfilled, (status, action) => {
  if (action.payload) {
    // successfully logged out 
    status.isAuthenticated = false;;
  }
});
```

Reply:
Thanks. I've updated the protected pages as well.